cli/src/main/java/com/devonfw/tools/ide/commandlet/CreateCommandlet.java

```diff
@@ -0,0 +1,75 @@
+package com.devonfw.tools.ide.commandlet;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.io.FileAccess;
+import com.devonfw.tools.ide.process.ProcessContext;
+import com.devonfw.tools.ide.process.ProcessResult;
+import com.devonfw.tools.ide.property.StringProperty;
+
+import java.nio.file.Path;
+
+/**
+ * {@link Commandlet} to create a new IDEasy instance
+ */
+public class CreateCommandlet extends Commandlet {
+
+  private final StringProperty newInstance;
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public CreateCommandlet(IdeContext context) {
+
+    super(context);
+    addKeyword(getName());
+    newInstance = add(new StringProperty("", false, "newInstance"));
```

Comment:
`newInstance` sounds rather technically and will not really help the end-user what this property/argument is about.
One could also guess wrong that this is a flag set to `true` or `false` configuring if a new instance is created or not...
After we have clarified that the git urls to clone into workspaces are called `repository` we also agreed that this here can now be called `project` without being ambiguous. I am not sure if you had already been on the old teams group where this discussion took place with the excel sheet asking how to uniquely name the things on the different leveles like `IDE_ROOT`, `IDE_HOME` and the sub-folders in the workspaces.

Reply:
done, renamed to `project`